exercises/sum-of-multiples/.meta/solutions/sum_of_multiples.rb

```diff
@@ -1,10 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 class SumOfMultiples
   attr_reader :multiples
   def initialize(*multiples)
     @multiples = multiples
   end
 
-  def to(limit)
+  def take(limit)
```

Comment:
What inspired the method name change?

Take a limit is a different idea potentially than to a limit.

The "to" implies an exclusive limit, while take suggests an inclusive limit.  An example of this in English is "go to the wall" which implies do not go all the way into the wall, or on the wall, just approach it as close as you can.  The Readme is clear that we are to exclude the limit.